repoman/bin/repoman

```diff
@@ -44,7 +44,7 @@ try:
 	repoman_main(sys.argv[1:])
 except IOError as e:
 	if e.errno == errno.EACCES:
-		print("\nRepoman: Need user access")
+		print("\nRepoman: You don't have access permition to do this operation")
```

Comment:
This message is too vague, we really need to handle the error closer to where it occurs. We can simply raise the IOError here, since it's a bug if it's not handled closer to where it occurs.

Reply:
We can simply handle `portage.exception.PermissionDenied` here.

Reply:
Allowing IOError to raise will be a good thing, since it will expose other areas that need attention.